### ISSUE TITLE
Text above map page overlaps

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#5058]
**[Submitted to the original trac issue database at 7.41am, Wednesday, 4th December 2013]**

The text (traces, diaries, etc.) and boxes (edit, history, etc.) above of the map overlap when the browserscreen is too narrow to support them side by side. When the browserscreen is narrowed even more, the links to traces, diaries etc disappear to be replaced by a "more" dropdown box. 
This dropdown box needs to be shown earlier to prevent the overlap.
See attached screenshots.

**Comment by tomhughes** [comment]
**Author: TomH**
**[Added to the original trac issue at 8.34am, Wednesday, 4th December 2013]**

Duplicate of https://github.com/openstreetmap/openstreetmap-website/issues/559.

I'm currently waiting on more translations go through (so that Copyright & License updates to just Copyright) before assessing how much to change it by.
